svg-min/gridicons-resize.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/><g><title>gridicons-resize</title><path d="M13 4v2h3.59L6 16.59V13H4v7h7v-2H7.41L18 7.41V11h2V4h-7"/></g></svg>
```

Comment:
@jasmussen I noticed that in the newest version of AI, it includes the title in the svg when you export. It looks like the minification script does not explicitly strip out the title, so it gets included in the generated icons.